.github/workflows/check-submodule-pointers.yml

```diff
@@ -0,0 +1,25 @@
+name: Check submodule pointers
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
this is redundant.

Reply:
I think we still want this, to test all PRs against main, e.g. https://github.com/github/vscode-codeql-starter/pull/64.